django_graph_api/graphql/types.py

```diff
@@ -385,11 +385,26 @@ def __init__(self, object_type, **kwargs):
 
     @property
     def object_type(self):
+        def _str_to_class(s):
+            try:
+                module_name, class_name = s.rsplit('.', 1)
+            except ValueError:
+                return None
```

Comment:
It would be great to have this function raise an exception instead of returning None so that users are directed to this spot when debugging (rather than wherever the class is meant to be instantiated).

Reply:
Addressed in 01f77b2